tools/pathifier/pathifier.xml

```diff
@@ -0,0 +1,138 @@
+        <param name="min_std" type="text" value="0.4" label="Minimum of standard deviation to filter out low variable genes"
+               help="Use 'data' to use the minimum standard deviation of your data" />
+        <param name="min_exp" type="text" value="4" label="Minimum of gene expression to filter out low variable genes"
+               help="Use 'data' to use the minimum expression of your data" />
+        <param name="heatmap_cluster_cells" label="Cluster samples on heatmap" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" />
+        <param name="heatmap_cluster_pathways" label="Cluster pathways on heatmap" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" />
+        <param name="heatmap_show_cell_labels" label="Show sample labels on heatmap" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false" />
+        <param name="heatmap_show_pathway_labels" label="Cluster pathway labels on heatmap" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" />
+        <param label="Return log file of pathifier" name="log" type="select" >
+            <option value="no" selected="True">No</option>
+            <option value="yes">Yes</option>
+        </param>
+             
+    </inputs>
+    <outputs>
+        <data name="pds" format="tabular" label="Pathifier Deregulation Score (PDS) of ${on_string}" />
+        <data name="logfile" format="txt" label="Pathifier log file of ${on_string}" >
+            <filter>log == 'yes'</filter>
+        </data>
+        <data name="plot" format="pdf" label="Pathifier vizualization ${on_string}" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="sheffer.tsv" ftype="tabular"/>
+            <param name="genes" value="kegg_pathways.gmt" ftype="txt" />
+            <param name="is_normal" value="True" />
+            <param name="normals" value="normals.tsv" ftype="tabular" />
+            <param name="log" value="yes" />
+            <param name="attempts" value="100" />
+            <output name="logfile" file="sheffer.kegg.log" ftype="txt" compare="sim_size" />
+            <output name="pds" file="sheffer.kegg.tsv" ftype="tabular"/>
+            <output name="plot" file="plot.pdf" ftype="pdf" compare="sim_size" />
+        </test>
+    </tests>
+    <help>
+
+**What it does**
+
+Pathifier is an algorithm that infers pathway deregulation scores for each tumor sample on the basis of 
+expression data. This score is determined, in a context-specific manner, for every particular dataset
+and type of cancer that is being investigated. The algorithm  transforms gene-level information into
+pathway-level information, generating a compact and biologically relevant representation of each sample.
+
+For each pathway analysed, the dataset is projected into the pathway gene space through a Principal 
+Component analysis (PCA) and then compute the principal curve. All transcriptomes are projected on 
```

Comment:
and a principal curve is regressed from these points. Gene transcriptomes are then projected on